Makefile

```diff
@@ -18,11 +18,11 @@ package: clean release
 	gzip -f9 promptoglyph.tar
 	rm -r promptoglyph
 
-$(BUILD_DIR)/promptoglyph-path: promptoglyph-path.d help.d
+$(BUILD_DIR)/promptoglyph-path: promptoglyph-path.d systempath.d help.d
```

Comment:
If `systempath.d` is only used in Windows and Windows uses a separate build script, why are we mentioning it in the Makefile?